src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -762,13 +760,11 @@ def cleanup_release_install_namespace_if_exists():
 
 # DO NOT use this method for re-put scenarios. This method involves new NS creation for helm release. For re-put
 #  scenarios, brownfield scenario needs to be handled where helm release still stays in default NS
-def helm_install_release(resource_manager, chart_path, subscription_id, kubernetes_distro, kubernetes_infra,
-                         resource_group_name, cluster_name, location, onboarding_tenant_id, http_proxy, https_proxy,
-                         no_proxy, proxy_cert, private_key_pem, kube_config, kube_context, no_wait, values_file,
-                         cloud_name, disable_auto_upgrade, enable_custom_locations, custom_locations_oid,
-                         helm_client_location, enable_private_link, arm_metadata,
-                         onboarding_timeout=consts.DEFAULT_MAX_ONBOARDING_TIMEOUT_HELMVALUE_SECONDS,
-                         container_log_path=None):
+def helm_install_release(resource_manager, chart_path, subscription_id, kubernetes_distro, kubernetes_infra, resource_group_name,
```

Comment:
Fix the lines so that they do not exceed the max length of 120 characters.  Run both azdev style and azdev linter after your changes.

Reply:
How did you get the max length error? I did not get an error like that in azdev style. I got this error:
```
R0914: Too many local variables (40/25) (too-many-locals)
```
But these were there before this change.

Reply:
Um.  I ran azdev style and it noted that these lines were longer than 120 characters - got explicit error message for that.  In fact, in the past, CLI reviewers have explicitly called that out as well.  So, it is a good practice to keep it at that limit.

Reply:
Do you know how you are running `azdev style`? environment. I run `azdev style` in a Python virtual environment and I don't get any errors about needing 120 characters. I will make the 120 width change still, but it would be good to set up a consistent development experience.